### PR TITLE
Creation of the did-endpoint documentation

### PR DESCRIPTION
**Pull request by ndefokou**
@Christiantyemele pleas review this PR

### CODE REVIEW

File: did-endpoint/README.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
+# did-endpoint 
+## Overview
+The `did-endpoint` is a robust, flexible, and extendable framework for managing Decentralized Identifiers (DIDs)
+
+## Purpose
+The did-endpoint aims to simplify the management of DIDs by providing a centralized service that supports multiple DID methods
```
Comment (Christiantyemele) [inline did-endpoint/README.md:6]: I think you meant descentralized service instead.

File: did-endpoint/README.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+## Usage
+- **Turns an HTTP(S) URL into a did:web id:** 
+```rust
+can_url_to_did_web_id() -> Result<(), Box<dyn Error>> {
+        assert_eq!(
+            url_to_did_web_id("localhost:8080")?,
+            "did:web:localhost%3A8080",
+        );
+}
```
Comment (Christiantyemele) [inline did-endpoint/README.md:23]: This is a test actually the real function to change HTTP URL to did:web id is ```url_to_did_web_id``` use it instead

File: did-endpoint/README.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+        }),
+        prm: Parameters::default(),
+    }
+diddoc = gen_diddoc(
+            &storage_dirpath,
+            &server_public_domain,
+            authentication_key.clone(),
+            assertion_key.clone(),
+            agreement_key.clone(),
+        )
```
Comment (Christiantyemele) [inline did-endpoint/README.md:74]: Use let statements when creating the variables .

**Review by Christiantyemele — CHANGES_REQUESTED** [review]

File: did-endpoint/README.md (hunk 1 above)
Comment (ndefokou) [reply did-endpoint/README.md]: yes thanks it was an error

**Review by Christiantyemele — APPROVED** [review]
LGTM 👍

**Review by Christiantyemele — APPROVED** [review]
Nice

**Pull request merged**